ansible/roles/atmo-ephemeral-mount/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+MOUNT_SOURCE: 'vdb'
```

Comment:
Can we dynamically figure this out?

Reply:
Maybe, but in my experience, an instance launched with an ephemeral disk will always have the ephemeral disk at [`/dev/vdb`](https://docs.openstack.org/glance/pike/admin/troubleshooting.html).

One more thing I forgot about: if you launch an instance using a flavor that has no ephemeral disk (e.g. on Jetstream Cloud), then attach a volume, that volume will also appear as `/dev/vdb`, so on redeploy of the same instance, the `atmo-ephemeral-mount` role will probably fail with error (because `/deb/vdb` is already mounted at `/vol1` or whatever).

Pushing another commit to fix this in a moment.

Reply:
Fixed.